adapters/rubicon.go

```diff
@@ -156,10 +156,67 @@ func parseRubiconSizes(sizes []openrtb.Format) (primary int, alt []int, err erro
 	return
 }
 
-func (a *RubiconAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder *pbs.PBSBidder) (pbs.PBSBidSlice, error) {
-	rpReq := makeOpenRTBGeneric(req, bidder, a.FamilyName())
+func (a *RubiconAdapter) CallOne(ctx context.Context, req *pbs.PBSRequest, reqJSON bytes.Buffer) (result fbResult, err error) {
```

Comment:
I was thinking of moving all/most of the code for handling request splitting to the main adapter file, but thought it was out of scope here. Will centralize this type.